ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -382,3 +382,41 @@ if get_option('enable-openvino')
     include_directories: include_directories('.')
   )
 endif
+
+if get_option('enable-mediapipe')
+  cc = meson.get_compiler('cpp')
+
+  cmd = run_command('sh', '-c', 'echo $MEDIAPIPE_HOME')
+  MEDIAPIPE_HOME = cmd.stdout().strip()
+  message('MEDIAPIPE_HOME: ' + MEDIAPIPE_HOME)
+
```

Comment:
It would be great if error handling is here when there is no $MEDIAPIPE_HOME env variable.

Reply:
I've considered the best way to check the validation of the ENV...
Unfortunately, as I know, there is no function to check the value(string) length or check the file existence in meson.
For this reason, IMO the error printing of meson is the best way for now on, and I remain it as is. any idea to make it better?

Reply:
[ -d ] returns 0 if MEDIAPIPE_HOME is undefined. Is this behavior ok?

```
mzx@kohaku:/source/AutoDrv/NNStreamer$ [ -d /source ]
mzx@kohaku:/source/AutoDrv/NNStreamer$ echo $?
0
mzx@kohaku:/source/AutoDrv/NNStreamer$ [ -d ]
mzx@kohaku:/source/AutoDrv/NNStreamer$ echo $?
0
mzx@kohaku:/source/AutoDrv/NNStreamer$ [ -d /ax ]
mzx@kohaku:/source/AutoDrv/NNStreamer$ echo $?
1
mzx@kohaku:/source/AutoDrv/NNStreamer$ 
```

Reply:
it's not ok, obviously. I'll make it better.

Reply:
one more checker with `-z` has been added.